### ISSUE TITLE
Client information tab

### ISSUE THREAD

**Issue by gjaklevich** [bels/openCOP#41]
Add client information tab to the dashboard. We could replace wiki and it is one less window to open.

**Comment by bels** [comment]
I agree.  Visually where would you expect information like this to show up? (ie. How would you navigate to it?)  Also what information would you be looking to store?

**Comment by gjaklevich** [comment]
I will send you a picture of what I have in mind

**Comment by infinitydexton** [comment]
Justin, it might be worthwhile to get the client information from the CRM. I put the data into there this week and for future, I would guess that by the time they become a client, they will be in the CRM. From what I know, it has an open API so we should not have a problem getting the information from one system to the other. - Just an idea.

**Comment by bels** [comment]
All in good time.
